install.sh

```diff
@@ -157,6 +157,32 @@ else
    rpicamdirEsc=""
 fi
 
+install_gpac()
+{
+    # if gpac is already installed, do nothing
+    if `command -v gpac >/dev/null`; then
+        return
+    fi
+
+    # install via apt if available
+    if `sudo apt install -y gpac`; then
```

Comment:
The backticks/command substitution leads to these command's STDOUT being executed, which of course fails. Instead we want to check the return code of the commands themselves, no substitution => no backticks.
```suggestion
    if command -v gpac >/dev/null; then
        return
    fi

    # install via apt if available
    if sudo apt install -y gpac; then
```